tests/pipeline/PipelineCTracing.cpp

```diff
@@ -0,0 +1,109 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/PipelineC/PipelineC.h"
+#include "revng/PipelineC/Tracing/Private.h"
+#include "revng/PipelineC/Tracing/Trace.h"
+#include "revng/Support/TemporaryFile.h"
+
+#define BOOST_TEST_MODULE PipelineCTracing
+bool init_unit_test();
+#include "boost/test/unit_test.hpp"
+
+#include "revng/UnitTestHelpers/UnitTestHelpers.h"
+
+namespace rt = revng::tracing;
```

Comment:
Avoid defining namespace aliases.
```suggestion
using namespace revng::tracing;
```

Reply:
Fixed